hail/python/benchmark/hail/fixtures.py

```diff
@@ -29,7 +29,7 @@ def __download(data_dir, filename):
     # Note: the below does not work on batch due to docker/ssl problems
     # dest = os.path.join(data_dir, filename)
     # urlretrieve(url, dest)
-    subprocess.check_call(['curl', f'--output={data_dir / filename}', '-s', url])
+    subprocess.check_call(['curl', url, '-s', '--output', f'{data_dir / filename}'])
```

Comment:
This will succeed even if there is an HTTP error code, and it won't follow redirects if they exist.

This can be fixed with `-Lfs` rather than just `-s` (or if you wanted to be verbose, `--fail`, `--location`, `--silent` [aka `-s`]).

Reply:
Thanks very much!